src/webgpu/api/operation/resource_init/texture_zero_init.spec.ts

```diff
@@ -467,127 +423,197 @@ export abstract class TextureZeroInitTest extends GPUTest {
     }
     this.queue.submit([commandEncoder.finish()]);
   }
+}
 
-  static generateParams(readMethods: ReadMethod[]) {
+const paramsBuilder = params()
+  .combine(
+    poptions('readMethod', [
+      ReadMethod.CopyToBuffer,
+      ReadMethod.CopyToTexture,
+      ReadMethod.Sample,
+      ReadMethod.DepthTest,
+      ReadMethod.StencilTest,
+    ])
+  )
+  .combine(poptions('format', kUncompressedTextureFormats))
+  .expand(({ format }) => {
+    const info = kUncompressedTextureFormatInfo[format];
+    let aspects: { aspect: GPUTextureAspect }[] = [];
+    if (info.depth && info.stencil) {
+      aspects = [{ aspect: 'depth-only' }, { aspect: 'stencil-only' }];
+      return aspects;
+    }
+    aspects = [{ aspect: 'all' }];
```

Comment:
This loses separate coverage for 'depth-only' and 'all' when both would be valid (same for stencil). Should we keep that coverage? It might behave a bit differently since implementations could have an error where they treat them differently for the purpose of identifying which aspects need to be cleared.

Reply:
We should indeed keep that coverage. I'll add it back.